README.md

```diff
@@ -97,67 +113,93 @@ datadog.addLambdaFunctions([<LAMBDA_FUNCTIONS>])
 datadog.addForwarderToNonLambdaLogGroups([<LOG_GROUPS>])
 ```
 
+#### Go
+
+```go
+import (
+	"github.com/DataDog/datadog-cdk-constructs/dist/go/ddcdkconstruct"
+)
+datadog := ddcdkconstruct.NewDatadog(
+    stack,
+    jsii.String("Datadog"),
+    &ddcdkconstruct.DatadogProps{
+        NodeLayerVersion:      jsii.Number(<LAYER_VERSION>),
+        AddLayers:             jsii.Bool(<BOOLEAN>),
+        Site:                  jsii.String(<SITE>),
+        ApiKey:                jsii.String(os.Getenv("DD_API_KEY")),
+        // ...
+    })
+    datadog.AddLambdaFunctions(&[]interface{}{myFunction}, nil)
+    datadog.AddForwarderToNonLambdaLogGroups()
+```
+
 ## Source Code Integration
+
 [Source code integration](https://docs.datadoghq.com/integrations/guide/source-code-integration/) is enabled by default through automatic lambda tagging, and will work if:
 
-- The Datadog Github Integration is installed.
+- The Datadog Github integration is installed.
```

Comment:
Addressing the comment from documentation team
https://github.com/DataDog/datadog-cdk-constructs/pull/271#discussion_r1702365958